css/styles.css

```diff
@@ -0,0 +1,137 @@
+body {
+    margin: 0;
+}
+.topnav {
+    background-color:rgb(25, 50, 126);
+    overflow: hidden;
+    width: 100%;
+    height: 3em;
+}
+
+.topnav a {
```

Comment:
We do not apply style to html elements. (why? 1. its difficult to understand what exactly is being styled and 2. It may affect other elements  that are not meant to have this style)

Reply:
change all and give it meaningful names.

Reply:
Even those who are in nested elements right?

Reply:
yes